examples/tdengineTelemetryService/deployment/http-deviceshifu-deployment.yaml

```diff
@@ -33,4 +36,8 @@ spec:
         - name: deviceshifu-config
           configMap:
             name: http-configmap-0.0.1
+        - name: deviceshifu-secret
```

Comment:
what happens if user have multiple secrets to different sql?

Reply:
This is a potential problem (after the telemetry is separated from deviceshifu, it can be modified later), but Yuqi said that "It is not possible to have two different telemetry pushes to two different sql endpoints in a deviceShifu", https://github.com/Edgenesis/shifu/pull/421#discussion_r1049337672, so there should not be multiple secrets in the sql protocol?